client/src/components/Onboarding/Availability/Availability.tsx

```diff
@@ -43,69 +47,80 @@ const Availability = ({ handleSubmit }: Props): JSX.Element => {
+            id={day + '-checkbox-field'}
+            key={day}
+            value={day}
+            control={<Checkbox checked={days[day]} onChange={handleChange} name={day} />}
+            label={day + 's'}
+            labelPlacement="bottom"
+          />
+        </Box>
       );
     });
   }
 
   return (
-    <Formik
-     initialValues={{
-       hours: '',
-       days: '',
-     }}
-     isSubmitting={true}
-     isValidating={true}
-     validationSchema={Yup.object().shape({
-       hours: Yup.string().required('Please choose your hours.'),
-       days: Yup.object().required('Please select your days.'),
-     })}
-     onSubmit={handleSubmit}
-    >
-      {({handleSubmit, handleChange, touched, errors}) => (
-        <form onSubmit={handleSubmit}>
-          <div>
-        <div>Available Hours: </div>
-        <div>
-          <TextField
-            id="start-hours-field"
-            value={hours}
-            helperText={touched.hours ? errors.hours : ''}
-            error={ touched.hours && Boolean(errors.hours)}
-            variant="outlined"
-            type="time"
-            name="start"
-            onChange={handleChange}
-          />
-          <span>-</span>
-          <TextField
-            id="end-hours-field"
-            value={hours}
-            helperText={touched.hours ? errors.hours : ''}
-            error={ touched.hours && Boolean(errors.hours)}
-            variant="outlined"
-            type="time"
-            name="end"
-            onChange={handleChange}
-          />
-        </div>
-      </div>
-
-      <div>
-        <div>Available Days: </div>
-        <FormGroup row>{renderCheckBoxes(days)}</FormGroup>
-      </div>
-        </form>
-      )}
-      </Formik>
-    );
-      }
+    <Box mt={6} className={classes.root}>
+      <AppLogo />
+      <Box className={classes.formWrapper}>
+        <ProgressBar progressText={'Set your availability'} progressValue={100} />
+        <Box className={classes.formItemsWrapper}>
+          <Formik
+            initialValues={{
+              hours: '',
+              days: '',
+            }}
+            isSubmitting={true}
+            isValidating={true}
+            validationSchema={Yup.object().shape({
+              hours: Yup.string().required('Please choose your hours.'),
+              days: Yup.object().required('Please select your days.'),
+            })}
+            onSubmit={handleSubmit}
+          >
+            {({ handleSubmit, handleChange, touched, errors }) => (
+              <form onSubmit={handleSubmit}>
+                <Box mx={7} mt={2} ml={6}>
+                  Available Hours:
+                </Box>
+                <Box mx={6} mt={1} ml={6} className={classes.formItem}>
+                  <TextField
+                    id="start-hours-field"
+                    value={hours}
+                    helperText={touched.hours ? errors.hours : ''}
+                    error={touched.hours && Boolean(errors.hours)}
```

Comment:
`Boolean(errors.hours)` is hours are a string then this will always be truthy even if it's `"false"` or `"0"`

Reply:
Unless, `errors.hours` is null